opencti-platform/opencti-front/src/private/components/analysis/malware_analyses/MalwareAnalysisDetails.tsx

```diff
@@ -0,0 +1,206 @@
+import makeStyles from '@mui/styles/makeStyles';
+import { graphql, useFragment } from 'react-relay';
+import React, { FunctionComponent } from 'react';
+import Typography from '@mui/material/Typography';
+import Paper from '@mui/material/Paper';
+import Grid from '@mui/material/Grid';
+import Chip from '@mui/material/Chip';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import { ViewModuleOutlined } from '@mui/icons-material';
+import ListItemText from '@mui/material/ListItemText';
+import StixDomainObjectNestedEntities from '../../common/stix_domain_objects/StixDomainObjectNestedEntities';
+import { Theme } from '../../../../components/Theme';
+import { MalwareAnalysisDetails_malwareAnalysis$data, MalwareAnalysisDetails_malwareAnalysis$key } from './__generated__/MalwareAnalysisDetails_malwareAnalysis.graphql';
+import ExpandableMarkdown from '../../../../components/ExpandableMarkdown';
+import { useFormatter } from '../../../../components/i18n';
+
+const styles = makeStyles<Theme>((theme) => ({
+  paper: {
+    height: '100%',
+    minHeight: '100%',
+    margin: '10px 0 0 0',
+    padding: '15px',
+    borderRadius: 6,
+  },
+  chip: {
+    fontSize: 12,
+    lineHeight: '12px',
+    backgroundColor: theme.palette.background.accent,
+    color: theme.palette.text?.primary,
+    textTransform: 'uppercase',
+    borderRadius: '0',
+    margin: '0 5px 5px 0',
+  },
+}));
+
+const MalwareAnalysisDetailsFragment = graphql`
+    fragment MalwareAnalysisDetails_malwareAnalysis on MalwareAnalysis {
+        id
+        product
+        result_name
+        result
+        submitted
+        analysis_started
+        analysis_ended
+        version
+        configuration_version
+        analysis_engine_version
+        analysis_definition_version
+        modules
+        creators {
+            id
+            name
+        }
+        objectLabel {
+            edges {
+                node {
+                    id
+                    value
+                    color
+                }
+            }
+        }
+    }
+`;
+
+interface MalwareAnalysisDetailsProps {
+  malwareAnalysisFragment: MalwareAnalysisDetails_malwareAnalysis$key
+}
+
+const MalwareAnalysisDetails: FunctionComponent<MalwareAnalysisDetailsProps> = ({ malwareAnalysisFragment }) => {
+  const { t, fldt } = useFormatter();
+  const classes = styles();
+
+  const data: MalwareAnalysisDetails_malwareAnalysis$data = useFragment(MalwareAnalysisDetailsFragment, malwareAnalysisFragment);
```

Comment:
Can you import as ?
Additionnaly, not sure you need to type data, useFragment should do it for you.